dbaas/logical/static/js/credential.js

```diff
@@ -160,21 +160,31 @@
         $(document).on("click.add-credential", "#add-credential", function(e) {
             $("tbody", "#table-credentials").append(
                 "<tr class='credential'><td colspan='3'>" +
-                "<input type='text' placeholder='type username' maxlength='16' name='user' value='' />" +
+                "<input type='text' placeholder='type username' maxlength='16' id='user''name='user' value='' />" +
+
+                "<select class='span' id='privileges' required>"+
+                "<option value='Owner' selected='selected'>Owner</option>"+
+                "<option value='Read-Write' selected='selected'>Read-Write</option>"+
+                "<option value='Read-Only' selected='selected'>Read-Only</option>"+
+                "</select>"+
+
                 "<a href='#' class='save-new-credential btn btn-primary'>Save</a>" +
-                "</td></tr>");
+                "</td></tr></div>");
         });
 
         $(document).on("click.save-new-credential", ".save-new-credential", function(e) {
             var $insert_row = $(e.target).parent().parent(),
-                username = $("input", $insert_row).val();
+                username = $("#user").val();
```

Comment:
Be careful using generic Ids like #user and #credentials. Use css classes when u can :D